sonar/modules/projects/serializers/__init__.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+#
+# Swiss Open Access Repository
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Projects serializers."""
+
+from __future__ import absolute_import, print_function
+
+from invenio_records_rest.serializers.response import record_responsify, \
+    search_responsify
+
+from sonar.modules.documents.api import DocumentRecord
+from sonar.modules.organisations.api import OrganisationRecord
+from sonar.modules.serializers import JSONSerializer as _JSONSerializer
+from sonar.modules.users.api import current_user_record
+
+from ..marshmallow import ProjectSchemaV1
+
+
+class JSONSerializer(_JSONSerializer):
+    """JSON serializer for projects."""
+
+    def post_process_serialize_search(self, results, pid_fetcher):
+        """Post process the search results."""
+        if current_user_record:
```

Comment:
why not in the marshmallow?

Reply:
In marshmallow, I think we have no access to aggregations..